DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -1,27 +1,59 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Portals
 {
-    using System.Data;
-    using System.Xml;
-    using System.Xml.Schema;
+    using DotNetNuke.Abstractions.Urls;
 
+    /// <summary>
+    /// Portal Alias Info interface.
+    /// </summary>
     public interface IPortalAliasInfo
     {
-        //BrowserTypes BrowserType { get; set; }
-        string CultureCode { get; set; }
+        /// <summary>
+        /// Gets or sets the http alias.
+        /// </summary>
         string HTTPAlias { get; set; }
-        bool IsPrimary { get; set; }
-        int KeyID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal Alias ID.
+        /// </summary>
         int PortalAliasID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal ID.
+        /// </summary>
         int PortalID { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether if the portal is the primary portal.
+        /// </summary>
+        bool IsPrimary { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the poral is a redirect portal.
+        /// </summary>
         bool Redirect { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Browser Type.
+        /// </summary>
+        BrowserTypes BrowserType { get; set; }
```

Comment:
This member was commented out before, what was the reason to add it back?

Reply:
Ah, was it just commented out b/c the enum type wasn't available to this project?

Reply:
That is correct, the enum type was not in the abstraction project and it would have created a circular dependency. Since our pattern is creating a carbon-copy of the enums and ensuring each item maps to an int it is now safe to bring it over and handle the casts.